app/move/controllers/assign/base.js

```diff
@@ -13,6 +16,16 @@ class CreateAllocationBaseController extends CreateBaseController {
     next()
   }
 
+  setMoveSummary(req, res, next) {
+    const currentLocation = req.session.currentLocation
+    const moveSubset = pick(res.locals.move, ['to_location', 'date'])
```

Comment:
I realised that this is to strip out the `move_agreed` field, right? As this is set to `false` even though technically is hasn't been answered yet. 

This is covering up that that is infact more of a bug with the API. We started to look to address this as part of prison to court as it should really be set to `null` if it wasn't provided with `POST /moves`. 

This fix also now prevents the answer showing up when the user actually answers that step in this journey. 

I think we should maybe wrap this in a TODO so that we know it needs to be addressed once we solve it in the API.

Reply:
You have inferred it correctly. I will add a TODO (and amend the commit wording)

Reply:
I wonder if actually we should be using `omit` instead of `pick`? That way any other future attributes from the move that the presenter needs to handle display it will still receive?

Reply:
To be honest, I intentionally went for `pick` to avoid unintentionally displaying other things that may be added in the future. The requirements here call for a specific set of properties displayed, not for `everything but`.

Reply:
But that's more the responsibility of the presenter than this method I'd say. That's why we only destructure particular properties within the presenter so that we're not displaying everything.

Let's leave this for now. I've raised [a ticket](https://dsdmoj.atlassian.net/browse/P4-1608) to address this issue in the correct place.

We will need to ensure this is reverted to correctly display the answer to the move agreed after it has been answered during this journey to consistently follow the pattern we have used elsewhere in the service.